ansible/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.8-slim-buster
+LABEL vendor="Binbash Leverage (info@binbash.com.ar)"
+
+ARG DOCKER_TAG
+ARG ANSIBLE_VERSION=${DOCKER_TAG}
+
+ENV PY_PIP_VER=20.2.4
+ENV PY_PIP_PKGS="ansible==${ANSIBLE_VERSION}"
+
+#
+# Installing dependencies
+#
+RUN apt-get update \
```

Comment:
@exequielrafaela Just something to discuss here: it is recommended not to perform a package update unless we then have a tight control over the versions of the packages we install (and its dependencies). Although I agree with this approach because it is more deterministic, I am bringing this up merely as a discussion starter since there are pros and cons to this approach as with any other one. Let me know what you think. 😉

Reply:
@diego-ojeda-binbash OJ thanks for your feedback here, I've faced the issue described here https://stackoverflow.com/questions/29929534/docker-error-unable-to-locate-package-git and that was why I've finally added the `RUN apt-get update` here. I'll take down note about this matter in order to review it during our next planning meeting! 🤝 ✅

Reply:
🤝

Reply:
I've used the word `here` quiet recurrently up there... sorry about that 😅